seqr/views/utils/orm_to_json_utils_tests.py

```diff
@@ -1,13 +1,13 @@
 from django.contrib.auth.models import User
 import mock
 from copy import deepcopy
-from seqr.models import Project, Family, Individual, Sample, IgvSample, SavedVariant, VariantNote, LocusList, VariantSearch
-from seqr.views.utils.orm_to_json_utils import get_json_for_user, _get_json_for_project, _get_json_for_family, \
-    _get_json_for_individual, get_json_for_sample, get_json_for_saved_variants, get_json_for_variant_note, get_json_for_locus_list, \
-    get_json_for_saved_search, get_json_for_saved_variants_with_tags, get_json_for_current_user
+from seqr.models import Project, Sample, IgvSample, SavedVariant, VariantNote, LocusList, VariantSearch
+from seqr.views.utils.orm_to_json_utils import get_json_for_user, _get_json_for_project, \
+    get_json_for_sample, get_json_for_saved_variants, get_json_for_variant_note, get_json_for_locus_list, \
+    get_json_for_saved_searches, get_json_for_saved_variants_with_tags, get_json_for_current_user
```

Comment:
You may want to add a test for `get_json_for_queryset` because there is an uncovered case of when `'guid' in value_fields`.

Reply:
I hate that we even have this test file. Utility functions should not, as a general rule, be tested on their own, they should be tested through the real, user-accessible functions that call them. There is no value to testing that helper functions stay the same, if the actual responses stays the same it shouldn't matter that some of the intermediate processing changed.  In the interest of maintaining coverage I'm not just deleting this file outright, but I am trying to move the test coverage from this file into the API tests. If there is a specific change you are worried about that you think we lost coverage for, let me know and I will add. test for it in the relevant api test

Reply:
In the case of `'guid' in value_fields` thats actually just a piece of duplicated code that needs to be cleaned up, so that is a good catch. But adding a utility function test to hit that code path would not have been the right fix - when a utility function is lacking in coverage the solution is to find where its being used and add a test for that, or remove the code path entirely if theres no case where it is hit